dsb-gui/src/main/resources/lang_en.properties

```diff
@@ -352,6 +352,7 @@ networkConError=Network connection error
 noTaskRunning=No tasks running
 filesAndFolders=files/folders have been loaded inside
 eonbrowser=Eon Browser
+buildVersion=1.0.0
```

Comment:
i removed this because buildVersion should be acquired from the BuildInfoService, which pulls the version from the jar